src/Language/Mulang/Parsers/Python.hs

```diff
@@ -73,10 +73,22 @@ muStatement (Print _ exprs _ _)               = M.Print $ compactMap muExpr expr
 muStatement (Exec expr _ _)                   = muExpr expr
 muStatement e                                 = M.debug e
 
+muClass (Just "unittest.TestCase") name body = M.TestGroup (M.MuString name) $ normalizeTests body
+muClass parent name                     body = M.Class name parent body
+
+normalizeTests (M.Sequence exprs) = M.Sequence $ map normalizeTest exprs
+normalizeTests expr               = normalizeTest expr
+
+normalizeTest func@(M.SimpleProcedure name _ body) = case isPrefixOf "test_" name of
+                                                      True  -> M.Test (M.MuString name) body
```

Comment:
I'd prefer to use guards here or even a local function

Reply:
Or an `if` expression

Reply:
yeah, that actually makes more sense. Whoops!